### PR TITLE
W-16176749-buildLogs&images-errors-DW12-dm

### PR DESCRIPTION
**Pull request by dmerlob**
Fixing Build Logs

### CODE REVIEW

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ The top-level definition in an FFD document starts with the form of the schema,
 
 If you are only working with one type of record, you only need to have a segment definition for that record type in your FFD.
 
-image::dataweave-flat-file-schemas-22788.png[]
+image::dataweave-flat-file-schemas-22788.png["A diagram showing the structure of a flat file schema with multiple elements and segments."]
```
Comment (hannanelson) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]: Do you need the "A diagram" bits? What about just "The structure of..."?

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 2]
```diff
@@ -129,7 +129,7 @@ This example contains 3 segments, each with a list of defined elements . It then
 
 If you have multiple structures or segment definitions in an FFD, when you apply your schema to an metadata description on a Transform Message component you'll need to specify which one you want to use.
 
-image::dataweave-flat-file-schemas-4e1eb.png[]
+image::dataweave-flat-file-schemas-4e1eb.png["A flat file schema showing groups, segments, and elements."]
```
Comment (hannanelson) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]: Like this...

Comment (hannanelson) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-4e1eb.png["A flat file schema showing groups, segments, and elements"]
```

Comment (hannanelson) [reply modules/ROOT/pages/dataweave-flat-file-schemas.adoc]: Global: Remove terminating punctuation for sentence fragments.

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc (hunk 1 above)
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-22788.png["A structure of a flat file schema with multiple elements and segments"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 3]
```diff
@@ -90,7 +90,7 @@ The simplified forms are only for convenience. You can use the _segments_ key ev
 
 If you're working with multiple types of records in the same transform, you'll need to use a structure definition that controls how these different records are combined.
 
-image::dataweave-flat-file-schemas-680c6.png[]
+image::dataweave-flat-file-schemas-680c6.png["A diagram showing the structure of a flat file schema with elements and segments."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-680c6.png["A structure of a flat file schema with elements and segments"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 4]
```diff
@@ -724,28 +724,29 @@ Here's a quick step by step example of how – once you have built out your full
 . In Anypoint Studio click *File* > *New* > *Mule Project* to create a new project, give it whatever name you wish and click *Finish*.
 . Drag an HTTP Connector from the palette to your empty canvas, then a Transform Message Component.
 +
-image::dataweave-flat-file-schemas-3f67f.png[]
+image::dataweave-flat-file-schemas-3f67f.png["A data transformation process involving HTTP and error handling."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-3f67f.png["A data transformation process involving HTTP and error handling"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 5]
```diff
@@ -724,28 +724,29 @@ Here's a quick step by step example of how – once you have built out your full
...
 
 
 
 . Click on the HTTP Connector to open its properties editor, then click the green arrow icon to create a new configuration for it. Leave all of the fields in default and click *OK*.
 +
-image::dataweave-quickstart-a56f0.png[]
+image::dataweave-quickstart-a56f0.png["An error message indicating a missing 'config-ref' attribute in an HTTP configuration screen."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-quickstart-a56f0.png["An error message indicating a missing 'config-ref' attribute in an HTTP configuration screen"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 6]
```diff
@@ -724,28 +724,29 @@ Here's a quick step by step example of how – once you have built out your full
...
 
 . In the *path* field write `get-copybook`. Once deployed, this will make the endpoint reachable through '+0.0.0.0:8081/get-copybook+'
 . Select the *Metadata* tab and click the *Set Metadata* button.
 +
-image::dataweave-quickstart-702fd.png[]
+image::dataweave-quickstart-702fd.png["A metadata configuration screen in a data transformation tool with an option to add metadata."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-quickstart-702fd.png["A metadata configuration screen in a data transformation tool with an option to add metadata"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 7]
```diff
@@ -724,28 +724,29 @@ Here's a quick step by step example of how – once you have built out your full
...
+
 . Click the *Edit* icon that appears next to the Payload element
 . Click the green plus icon to add a new metadata type and name it 'my-flat-file-type'
 
 . Select *Flat File* as the type, on the *Schema* field point it to the location of your schema file. Then click *Select* to use this Metadata type.
 +
-image::dataweave-flat-file-schemas-814e1.png[]
+image::dataweave-flat-file-schemas-814e1.png["A selection screen for flat file metadata types with details on various record structures."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-814e1.png["A flat file metadata types screen with details on various record structures"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 8]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
 . If you haven't already, download and extract the link:{attachmentsdir}/dw-quickstart-bundle.zip[dw-quickstart-bundle.zip] and drag both files in it to the `src/main/resources` folder of your new Mule project.
 . Drag an xref:3.9@mule-runtime::http-connector.adoc[HTTP Connector] from the palette onto your empty canvas, followed by a *Transform Message* component, then another HTTP Connector.
 +
-image::dataweave-quickstart-5a4cb.png[]
+image::dataweave-quickstart-5a4cb.png["A schematic view of a data transformation process."]
 . Click on the first HTTP Connector to open its properties editor, then click the green plus sign icon to create an HTTP connector configuration. In the popup, leave the default values for all fields--just click *OK*.
 +
-image::dataweave-quickstart-a56f0.png[]
+image::dataweave-quickstart-a56f0.png["An error message indicating a missing 'config-ref' attribute in an HTTP configuration screen."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-a56f0.png["An error message indicating a missing 'config-ref' attribute in an HTTP configuration screen"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 9]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 
 . In the *Path* field in the properties window above, write `place-order`. Once the Mule application is deployed, the desired endpoint becomes reachable at `+0.0.0.0:8081/place-order+`
 . Select the *Metadata* tab from the HTTP connector's properties window and click the *Add Metadata* button.
 +
-image::dataweave-quickstart-702fd.png[]
+image::dataweave-quickstart-702fd.png["A metadata configuration screen in a data transformation tool with an option to add metadata."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-702fd.png["A metadata configuration screen in a data transformation tool with an option to add metadata"]
```

File: modules/ROOT/pages/dataweave-flat-file-schemas.adoc [hunk 10]
```diff
@@ -724,28 +724,29 @@ Here's a quick step by step example of how – once you have built out your full
...
 
 . Note that if you open the Transform Message component, the input section should show the fields that will be present in the incoming payload.
 +
-image::dataweave-flat-file-schemas-a677a.png[]
+image::dataweave-flat-file-schemas-a677a.png["A data transformation interface showing flat file header configurations and output preview."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-flat-file-schemas.adoc]:
```suggestion
image::dataweave-flat-file-schemas-a677a.png["A data transformation interface showing flat file header configurations and output preview"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 11]
```diff
@@ -158,7 +158,7 @@ You can set theses properties through the XML of your Mule project:
 
 Alternatively, you can set the properties through the UI of the Transform Message component:
 
-image::dataweave-formats-580be.png[]
+image::dataweave-formats-580be.png["A reader configuration screen showing the different properties to be set."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-580be.png["A reader configuration screen showing the different properties to be set"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 12]
```diff
@@ -197,7 +197,7 @@ In the Transform Message component, you can define a CSV type through the follow
 * Via a graphical editor that allows you to set up each field manually
 
 +
-image::dataweave-formats-4a556.png[]
+image::dataweave-formats-4a556.png["A metadata type selection screen showing options to define CSV and XML data types."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-4a556.png["A metadata type selection screen showing options to define CSV and XML data types"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 13]
```diff
@@ -215,7 +215,7 @@ For example:
 
 .Input
 
-image::dataweave-formats-exceltable.png[]
+image::dataweave-formats-exceltable.png["A table in an Excel format displaying IDs and names starting from row 1."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-exceltable.png["A table in an Excel format displaying IDs and names"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 14]
```diff
@@ -280,7 +280,7 @@ These properties can be either set via the XML of your Mule project:
 
 Or via the UI of the Transform Message component:
 
-image::dataweave-formats-excell-reader.png[]
+image::dataweave-formats-excell-reader.png["A reader configuration dialog for Excel with options for handling empty lines and setting table offsets."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-excell-reader.png["A reader configuration dialog for Excel with options for handling empty lines and setting table offsets"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 15]
```diff
@@ -313,7 +313,7 @@ In the Transform Message component, you can define a excel type through the foll
 * Via a graphical editor that allows you to set up each field manually
 
 +
-image::dataweave-formats-excel-metadata.png[]
+image::dataweave-formats-excel-metadata.png["A Select metadata type screen with an option to select Type and a table listing fields like name, address, and age."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-excel-metadata.png["A select metadata type screen with an option to select Type and a table listing fields like name, address, and age"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 16]
```diff
@@ -393,7 +393,7 @@ These properties can be either set via the XML of your Mule project:
 
 Or via the UI of the Transform Message component:
 
-image::dataweave-formats-6e5e4.png[]
+image::dataweave-formats-6e5e4.png["A reader configuration screen with options for handling null values and optimizing for speed."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-6e5e4.png["A reader configuration screen with options for handling null values and optimizing for speed"]
```

File: modules/ROOT/pages/dataweave-formats.adoc [hunk 17]
```diff
@@ -677,7 +677,7 @@ In the Transform Message component, you can define a Fixed Width type through th
 * Via a graphical editor that allows you to set up each field manually
 
 +
-image::dataweave-formats-27b3c.png[]
+image::dataweave-formats-27b3c.png["A metadata selection screen with various data type configurations."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-formats.adoc]:
```suggestion
image::dataweave-formats-27b3c.png["A metadata selection screen with various data type configurations"]
```

File: modules/ROOT/pages/dataweave-memory-management.adoc [hunk 18]
```diff
@@ -49,7 +49,7 @@ Below is an example that sets this attribute to "deferred":
 
 Keep in mind that when using deferred execution, the returned message will contain a `WeaveOutputHandler` object rather than a String representation. For example, consider a logger you wish to log your payload with:
 
-image::dataweave-memory-management-541a1.png[]
+image::dataweave-memory-management-541a1.png["A flow diagram showing steps from reading CSV to writing JSON."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-memory-management.adoc]:
```suggestion
image::dataweave-memory-management-541a1.png["A flow diagram showing steps from reading CSV to writing JSON"]
```

File: modules/ROOT/pages/dataweave-memory-management.adoc [hunk 19]
```diff
@@ -65,7 +65,7 @@ org.mule.transport.file.FileConnector: Writing file to: /Users/mulesoft/output/i
 
 If you wish to log the payload as a String representation, you'll need to request the payload in a String representation. This can be achieved by using the expression `#[message.payloadAs(java.lang.String)]`.
 
-image::dataweave-memory-management-dc180.png[]
+image::dataweave-memory-management-dc180.png["A flow diagram for handling JSON output in a data processing sequence."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-memory-management.adoc]:
```suggestion
image::dataweave-memory-management-dc180.png["A flow diagram for handling JSON output in a data processing sequence"]
```

File: modules/ROOT/pages/dataweave-migrator.adoc [hunk 20]
```diff
@@ -16,7 +16,7 @@ To migrate a DataMapper component to DataWeave, do the following:
 
 . Right click on a DataMapper component, select *Migrate to DataWeave*
 +
-image::dw-migrator-script.png[dw migrator]
+image::dw-migrator-script.png["A script interface for mapping worker types in a migration tool with options for user query and data storage."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-migrator.adoc]:
```suggestion
image::dw-migrator-script.png["A script interface for mapping worker types in a migration tool with options for user query and data storage"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 21]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
 . If you haven't already, download and extract the link:{attachmentsdir}/dw-quickstart-bundle.zip[dw-quickstart-bundle.zip] and drag both files in it to the `src/main/resources` folder of your new Mule project.
 . Drag an xref:3.9@mule-runtime::http-connector.adoc[HTTP Connector] from the palette onto your empty canvas, followed by a *Transform Message* component, then another HTTP Connector.
 +
-image::dataweave-quickstart-5a4cb.png[]
+image::dataweave-quickstart-5a4cb.png["A schematic view of a data transformation process."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-5a4cb.png["A data transformation process"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 22]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 
 . In the *Path* field in the properties window above, write `place-order`. Once the Mule application is deployed, the desired endpoint becomes reachable at `+0.0.0.0:8081/place-order+`
 . Select the *Metadata* tab from the HTTP connector's properties window and click the *Add Metadata* button.
 +
-image::dataweave-quickstart-702fd.png[]
+image::dataweave-quickstart-702fd.png["A metadata configuration screen in a data transformation tool with an option to add metadata."]
 . Click the *Edit* icon that appears next to the dropdown.
 +
-image::dataweave-quickstart-8ea02.png[]
+image::dataweave-quickstart-8ea02.png["An error message on an HTTP configuration screen indicating a missing attribute."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-8ea02.png["An error message on an HTTP configuration screen indicating a missing attribute"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 23]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 +
 . Click the green plus sign to *Add* a new metadata type.
 +
-image::dataweave-quickstart-c8ae5.png[]
+image::dataweave-quickstart-c8ae5.png["A user interface for selecting metadata types with options to add or delete types."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-c8ae5.png["A user interface for selecting metadata types with options to add or delete types"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 24]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 +
 . Name the metadata type `XML-order-input` and click *Create type*.
 +
-image::dataweave-quickstart-22a00.png[]
+image::dataweave-quickstart-22a00.png["A user interface for creating a new XML metadata type."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-22a00.png["A user interface for creating a new XML metadata type"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 25]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 +
 . Select *XML* as the type, choose *Example* from the dropdown and point it to the location of the `dw-tshirt-inputsample.xml` file, which you can extract from the bundle you downloaded at the start of this guide (link:{attachmentsdir}/dw-quickstart-bundle.zip[dw-quickstart-bundle.zip]). Then click *Select* to use this metadata type.
 +
-image::dataweave-quickstart-26f37.png[]
+image::dataweave-quickstart-26f37.png["A metadata type selection interface for XML data related to orders, showing fields like buyer and address."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-26f37.png["A metadata type selection interface"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 26]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 +
 [TIP]
 ====
 Now if you select the Transform Message component by clicking it in the flow, the input section should show the fields that are expected in the *incoming payload* through the Transform Message.
 
-image::dataweave-quickstart-4a1db.png[]
+image::dataweave-quickstart-4a1db.png["A data transformation interface showing input fields for product details and output in JSON format."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-4a1db.png["A data transformation interface showing input fields for product details and output in JSON format"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 27]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 ====
 . Click on the *_second HTTP connector in the flow_* to open its properties editor. Click the green plus sign to create a new configuration for the HTTP connector. In the Global Element Properties popup, change the Port field to 8082 (it must be changed since port 8081 is already taken by the other HTTP connector).
 . In the global configuration popup, find the "RAML Location" field, and give the proper path to the `t-shirt.raml` file, which is available in the bundle you downloaded earlier. You can either type in the file location, or – if you extracted the file into the Mule project folder – locate it through the *browse* button. Then click *Ok*.
 +
-image::dataweave-quickstart-6d4b7.png[]
+image::dataweave-quickstart-6d4b7.png["A configuration screen for an HTTP request with settings for host, port, and base path."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-6d4b7.png["An HTTP request configuration interface with settings for host, port, and base path"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 28]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 
 . Back in the HTTP connector properties window, set "Path" to */orders* and use *POST* as the "Method", picking out of the options described in the RAML you attached.
 +
-image::dataweave-quickstart-3c9c7.png[]
+image::dataweave-quickstart-3c9c7.png["A configuration screen for an HTTP request with fields for URL settings and method POST."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-3c9c7.png["A configuration screen for an HTTP request with fields for URL settings"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 29]
```diff
@@ -96,52 +96,52 @@ It is always recommended you set up the other components of your flow before you
...
 
 +
 [TIP]
 ====
 Note that if you click on the Transform Message component, the output section now reflects the data structure that's described in the RAML file for *POST*-ing to `/orders`.
 
-image::dataweave-quickstart-f39a5.png[]
+image::dataweave-quickstart-f39a5.png["A data transformation interface showing XML input and JSON output configurations."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-f39a5.png["A data transformation interface showing XML input and JSON output configurations"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 30]
```diff
@@ -155,12 +155,12 @@ You can create your DataWeave mapping without going through any of this, just ke
 
 . On the package explorer, left click on the `t-shirt.raml` file in your project and select *Mule* > *Generate Flows from RAML*. This automatically generates a new Mule configuration XML that includes a Mule flow per each resource and method described on the RAML.
 +
-image::dataweave-quickstart-flows-from-raml.png[]
+image::dataweave-quickstart-flows-from-raml.png["A project directory structure for a DataWeave application with options for managing metadata."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-flows-from-raml.png["A Mule configuration XML generated from RAML"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 31]
```diff
@@ -176,7 +176,7 @@ Use the xref:6.x@studio::transform-message-component-concept-studio.adoc[About T
 
 *Note:* The `address2` and `size` fields in the output side should remain unassigned.
 
-image::dataweave-quickstart-1984d.png[map]
+image::dataweave-quickstart-1984d.png["A Transform Message component showing a mapping between the input and output."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-1984d.png["A Transform Message component showing a mapping between the input and output"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 32]
```diff
@@ -155,12 +155,12 @@ You can create your DataWeave mapping without going through any of this, just ke
...
 . On the package explorer, left click on the `t-shirt.raml` file in your project and select *Mule* > *Generate Flows from RAML*. This automatically generates a new Mule configuration XML that includes a Mule flow per each resource and method described on the RAML.
 +
-image::dataweave-quickstart-flows-from-raml.png[]
+image::dataweave-quickstart-flows-from-raml.png["A project directory structure for a DataWeave application with options for managing metadata."]
 +
 . Open the `t-shirt.xml` file that was generated on your project and look for the flow labeled `post:/orders:application/json:t-shirt-config`.
 . Open the properties for the `Set Payload` element in that flow, and set its value to `&#x0023;[payload]`
 +
-image::dataweave-quickstart-set-payload.png[]
+image::dataweave-quickstart-set-payload.png["A configuration screen for setting payload in a message flow with MIME type settings."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-set-payload.png["A configuration screen for setting payload in a message flow"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 33]
```diff
@@ -202,11 +202,11 @@ The transform that we mapped in the previous section does not account for the fi
 
 . Double click on the `address2` field in the output, seeing how this adds an icon next to the `address2` field, and generates a line of DataWeave code that assigns this field the value `null`.
 +
-image::dataweave-quickstart-c1e53.png[]
+image::dataweave-quickstart-c1e53.png["A data transformation interface detailing fields for input and output information in JSON format."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-c1e53.png["A Transform Message interface detailing fields for input and output information in JSON format"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 34]
```diff
@@ -202,11 +202,11 @@ The transform that we mapped in the previous section does not account for the fi
...
 +
 . Do the same for the `size` field, then edit the DataWeave code directly to assign `size` the value `M`, for example, rather than going with the `null` value.
 +
-image::dataweave-quickstart-4abf1.png[]
+image::dataweave-quickstart-4abf1.png["A data transformation interface showing fields for input and output information in JSON format."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-4abf1.png["A Transform Message interface showing fields for input and output information in JSON format"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 35]
```diff
@@ -243,15 +243,15 @@ The Transform Message component provides realtime feedback in the editor; a prev
 
 . To open this section click the `Preview` button on the top right corner of the editor.
 +
-image::dataweave-quickstart-07f8a.png[]
+image::dataweave-quickstart-07f8a.png["Preview button icon", 5%, 5%]
 +
 . As your metadata is based off a sample input you provided before, Studio uses the data in this sample to build out the output sample.
 +
-image::dataweave-quickstart-59e9c.png[]
+image::dataweave-quickstart-59e9c.png["A JSON output preview mapping output's details like address and email from an order."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-59e9c.png["A JSON output preview mapping output's details"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 36]
```diff
@@ -243,15 +243,15 @@ The Transform Message component provides realtime feedback in the editor; a prev
...
 +
 . You can freely edit the sample data to test what would happen in corner cases. Select the `payload` tab in your input section and replace any values there at will. When doing this, note how these values are populated into the values of the preview in real time, giving you a tangible representation of what the final result of your mapping will be.
 +
-image::dataweave-quickstart-75ca5.png[]
+image::dataweave-quickstart-75ca5.png["A snippet of an XML input sample."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-75ca5.png["A snippet of an XML input sample"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 37]
```diff
@@ -292,7 +292,7 @@ You should get a response with an JSON body that has a single value, this is the
 
 . Drag two HTTP connectors and a Transform Message component into a new flow.
 +
-image::dataweave-quickstart-e4466.png[]
+image::dataweave-quickstart-e4466.png["A flow of a data transformation process involving HTTP and error handling."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-e4466.png["A flow of a data transformation process"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 38]
```diff
@@ -303,12 +303,12 @@ image::dataweave-quickstart-e4466.png[]
 . Set the Path to */products* and the Method to *GET*.
 . If you go and click the Transform Message component, note that its input has been populated with the appropriate data structure to send, via a GET request to /products.
 +
-image::dataweave-quickstart-2196b.png[]
+image::dataweave-quickstart-2196b.png["A user interface for transforming message data with a focus on output payload structure."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-2196b.png["A transforming message data with a focus on output payload structure"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 39]
```diff
@@ -303,12 +303,12 @@ image::dataweave-quickstart-e4466.png[]
...
 
 . Click the *Preview* button to open the output preview. As there is no sample data available, you will be prompted to provide a sample.
 
 +
-image::dataweave-quickstart-be7fd.png[]
+image::dataweave-quickstart-be7fd.png["A preview screen prompting the creation of sample data for execution."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-be7fd.png["A preview screen prompting the creation of sample data for execution"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 40]
```diff
@@ -349,24 +349,24 @@ As with the other Transform Message component used in the first example of this
 
 Click your Transform Message component. In the text editor pane, replace the curly brackets '{}' for 'payload'.
 
-image::dataweave-quickstart-7c173.png[]
+image::dataweave-quickstart-7c173.png["An output payload preview in a data transformation tool."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-7c173.png["An output payload preview in a data transformation tool"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 41]
```diff
@@ -349,24 +349,24 @@ As with the other Transform Message component used in the first example of this
...
 
 This is the simplest transformation you can carry out, it replicates the input structure but changes the format from *JSON* to *Java*. Whatever exists in the payload – including any child elements at any depth – is transformed directly into Java without changing any of its structure.
 
 Open the *Preview* tab and see a tree describing the structure of the data, this is what a Java preview always looks like:
 
-image::dataweave-quickstart-fb3b7.png[]
+image::dataweave-quickstart-fb3b7.png["A Java output payload displaying a list of attributes."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-fb3b7.png["A Java output payload displaying a list of attributes"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 42]
```diff
@@ -349,24 +349,24 @@ As with the other Transform Message component used in the first example of this
...
 
 You can easily change the xref:dataweave-language-introduction.adoc#output-directive[output directive] from the default `application/java` to `text/csv` or to any other of the xref:dataweave-formats.adoc[supported formats] and see the results of that transformation.
 
-image::dataweave-quickstart-c8b4f.png[]
+image::dataweave-quickstart-c8b4f.png["A CSV output preview."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-c8b4f.png["A CSV output preview"]
```

File: modules/ROOT/pages/dataweave-quickstart.adoc [hunk 43]
```diff
@@ -349,24 +349,24 @@ As with the other Transform Message component used in the first example of this
...
 
 [NOTE]
 Note that in this particular example, the format `application/XML` won't work as is because XML requires a parent node, which is absent here.
 
 Change the xref:dataweave-language-introduction.adoc#output-directive[output directive] to `application/json` so that the final result is identical to the sample data in the input. Later on you can transform this in more interesting ways.
 
-image::dataweave-quickstart-6381c.png[]
+image::dataweave-quickstart-6381c.png["A JSON output preview."]
```
Comment (dmerlob) [inline modules/ROOT/pages/dataweave-quickstart.adoc]:
```suggestion
image::dataweave-quickstart-6381c.png["A JSON output preview"]
```

**Review by Crispy-Salesforce — APPROVED** [review]
